packages/sources/peregrine-fund-admin/src/endpoint/nav.ts

```diff
@@ -0,0 +1,35 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { config } from '../config'
+import overrides from '../config/overrides.json'
+import { httpTransport } from '../transport/nav'
+
+// Input parameters define the structure of the request expected by the endpoint. The second parameter defines example input data that will be used in EA readme
+export const inputParameters = new InputParameters({
+  assetId: {
+    aliases: ['assetId'],
```

Comment:
This can be removed

Reply:
If i remove this what does the BaseEndpointTypes Parameters value become?

// Endpoints contain a type parameter that allows specifying relevant types of an endpoint, for example, request payload type, Adapter response type and Adapter configuration (environment variables) type
export type BaseEndpointTypes = {
  Parameters: typeof inputParameters.definition
  Response: SingleNumberResultResponse
  Settings: typeof config.settings
}

Reply:
I mean 
```
aliases: ['assetId'],
```
Because it is the same name, so you don't have to specify it again

Reply:
Ah ok just the alias not the InputParameter object itself, thank you.